src/Functions.Worker.Extensions.OpenAI/Search/ChatCompletionJsonConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.AI.OpenAI;
+
+namespace WebJobs.Extensions.OpenAI.Search;
```

Comment:
worker namespaces should be:
`Microsoft.Azure.Functions.Worker.Extensions.OpenAI.Search`